recipes-devtools/upm/upm_0.7.1.bb

```diff
@@ -16,16 +16,14 @@ inherit distutils-base pkgconfig python-dir cmake
 CFLAGS_append_edison = " -msse3 -mfpmath=sse"
 
 FILES_${PN}-doc += " ${datadir}/upm/examples/"
-RDEPENDS_${PN} += " mraa"
+RDEPENDS_${PN} += " mraa mraa-java"
```

Comment:
@tripzero it's not an optional runtime dependency to the upm package as the recipe is today.  If the mraa-java package is NOT built and the upm package is configured to build java-upm, the upm build will fail.